htdocs/package.json

```diff
@@ -12,8 +12,9 @@
         "prepare": "npm run generate-css"
     },
     "dependencies": {
+        "bootstrap": "^5.1.3",
```

Comment:
No.

Reply:
I think I added that before my latest round of changes.  I'm trying to strip out as much html/css/js as possible.